src/styles/blocks/card-block/stars.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
```

Comment:
It's better to use absolute paths for assets like images instead of relative paths. Change url(../images/star.svg) to url(/images/star.svg)